zkevm-circuits/src/keccak_circuit/keccak_packed_multi.rs

```diff
@@ -17,6 +17,13 @@ pub(crate) fn get_num_rows_per_round() -> usize {
         .expect("Cannot parse KECCAK_ROWS env var as usize")
 }
 
+pub(crate) fn keccak_unusable_rows() -> usize {
+    const UNUSABLE_ROWS_BY_KECCAK_ROWS: [usize; 20] = [
+        21, 39, 41, 59, 57, 59, 65, 59, 53, 67, 63, 59, 45, 79, 77, 75, 73, 71, 69, 67,
+    ];
+    UNUSABLE_ROWS_BY_KECCAK_ROWS[get_num_rows_per_round() - 1]
+}
```

Comment:
Where does this rows info come from? Why are all keccak-rows different in terms of unusable rows?

Reply:
These number comes from different `num_rows_per_round` setting, still couldn't figure out a formula to define it, they are collected after running the `configure`.